src/models/companyHouseEntity.ts

```diff
@@ -78,6 +78,7 @@ export default class CompanyHouseEntity {
   async getCompanyProfileByCompanyNumber(companyNumber: string): Promise<CompanyProfileResult> {
     const endpoint = `${this.env.get('COMPANY_HOUSE_API_URL')}/company/${encodeURIComponent(companyNumber)}`
     const companyProfile = await this.makeCompanyProfileRequest(endpoint)
+    console.log({ companyProfile })
```

Comment:
remove console log please. Either remove it completely or replace it with a logger call if the message is useful